### ISSUE TITLE
Crash when starting game

### ISSUE THREAD

**Issue by FeelsBadMatey** [CleverRaven/Cataclysm-DDA#33217]
Title.

 - OS: Windows 7
 - Game Version: 9485 (unable to get exact version due to nature of problem)
 - Graphics version: Tiles
 - Mods loaded: Base game mods

# Additional context
CRASH LOG FILE: config/crash.log
VERSION: 0.D-6516-g278482c
TYPE: St13runtime_error
MESSAGE: locale::facet::_S_create_c_locale name not valid
STACK TRACE:
    @0x59DC95[cataclysm-tiles.exe+0x19DC95]
    @0x59E842[cataclysm-tiles.exe+0x19E842]
    IMG_LoadWEBP_RW+0x198A08@0x11BC938[cataclysm-tiles.exe+0xDBC938]
    IMG_LoadWEBP_RW+0x53EC73@0x1562BA3[cataclysm-tiles.exe+0x1162BA3]
    IMG_LoadWEBP_RW+0x54511E@0x156904E[cataclysm-tiles.exe+0x116904E]
    IMG_LoadWEBP_RW+0x531927@0x1555857[cataclysm-tiles.exe+0x1155857]
    IMG_LoadWEBP_RW+0x3E42F3@0x1408223[cataclysm-tiles.exe+0x1008223]
    IMG_LoadWEBP_RW+0x3E2311@0x1406241[cataclysm-tiles.exe+0x1006241]
    IMG_LoadWEBP_RW+0x3E465E@0x140858E[cataclysm-tiles.exe+0x100858E]
    @0xB6A9D9[cataclysm-tiles.exe+0x76A9D9]
    @0xB6B117[cataclysm-tiles.exe+0x76B117]
    IMG_LoadWEBP_RW+0x59BB70@0x15BFAA0[cataclysm-tiles.exe+0x11BFAA0]
    @0x4013ED[cataclysm-tiles.exe+0x13ED]
    @0x4014FB[cataclysm-tiles.exe+0x14FB]
    BaseThreadInitThunk+0xD@0x76EE652D[kernel32.dll+0x1652D]
    RtlUserThreadStart+0x21@0x7711C521[ntdll.dll+0x2C521]

-----------------------------------------
00:11:40.010 : Starting log.
00:11:40.010 INFO : Cataclysm DDA version 0.D-6516-g278482c
00:11:40.041 WARNING : opendir [./sound/] failed with "No such file or directory".
00:11:40.108 INFO : Number of render drivers on your system: 4
00:11:40.108 INFO : Render driver: 0/direct3d
00:11:40.109 INFO : Render driver: 1/direct3d11
00:11:40.109 INFO : Render driver: 2/opengl
00:11:40.109 INFO : Render driver: 3/software
00:11:50.989 : Log shutdown.
-----------------------------------------

**Comment by Shotgunface** [comment]
Just opened the same issue right before you should I close mine? (yours is in better format)
https://github.com/CleverRaven/Cataclysm-DDA/issues/33216#issue-480474088

**Comment by FeelsBadMatey** [comment]
> Just opened the same issue right before you should I close mine? (yours is in better format)
> [#33216 (comment)](https://github.com/CleverRaven/Cataclysm-DDA/issues/33216#issue-480474088)

I guess so, don't really need to clutter up the issue page

**Comment by SomeDeadGuy** [comment]
Same.
If I delete options.json game starts but once I change options and save them game crashes and no longer let me start the game.

**Comment by Alterforce** [comment]
I am also having this issue with build 9488.  I downloaded it via the launcher and it always crashes unless I delete options.json first.

**Comment by Qrox** [comment]
This should already be fixed by #33214.

**Comment by ampersand55** [comment]
I also get the error, build 9489.

```
CRASH LOG FILE: config/crash.log
VERSION: 0.D-6523-g9857661
TYPE: St13runtime_error
MESSAGE: locale::facet::_S_create_c_locale name not valid
STACK TRACE:
    @0x59DC95[cataclysm-tiles.exe+0x19DC95]
    @0x59E842[cataclysm-tiles.exe+0x19E842]
    IMG_LoadWEBP_RW+0x198A08@0x11BE0E8[cataclysm-tiles.exe+0xDBE0E8]
    IMG_LoadWEBP_RW+0x53F6C3@0x1564DA3[cataclysm-tiles.exe+0x1164DA3]
    IMG_LoadWEBP_RW+0x545B6E@0x156B24E[cataclysm-tiles.exe+0x116B24E]
    IMG_LoadWEBP_RW+0x5322C7@0x15579A7[cataclysm-tiles.exe+0x11579A7]
    IMG_LoadWEBP_RW+0x3E4443@0x1409B23[cataclysm-tiles.exe+0x1009B23]
    IMG_LoadWEBP_RW+0x3E2461@0x1407B41[cataclysm-tiles.exe+0x1007B41]
    IMG_LoadWEBP_RW+0x3E47AE@0x1409E8E[cataclysm-tiles.exe+0x1009E8E]
    @0xB6AAC9[cataclysm-tiles.exe+0x76AAC9]
    @0xB6B207[cataclysm-tiles.exe+0x76B207]
    IMG_LoadWEBP_RW+0x59C5C0@0x15C1CA0[cataclysm-tiles.exe+0x11C1CA0]
    @0x4013ED[cataclysm-tiles.exe+0x13ED]
    @0x4014FB[cataclysm-tiles.exe+0x14FB]
    BaseThreadInitThunk+0x14@0x7FFF910C84D4[KERNEL32.DLL+0x84D4]
    RtlUserThreadStart+0x21@0x7FFF9231E851[ntdll.dll+0x6E851]

```

**Comment by ampersand55** [comment]
Can confirm this is fixed as of #9490.
